terragrunt/modules/release-distribution/lambdas/doc-router/index.js

```diff
@@ -66,10 +66,16 @@ exports.handler = (event, context, callback) => {
         return temp_redirect('https://www.rust-lang.org/learn', callback);
     }
 
-    // Forward versioned documentation as-is.
-    if (/^\/\d/.test(request.uri)) {
+    // Forward patch versioned documentation as-is.
+    if (/^(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
```

Comment:
```suggestion
    if (/^\/(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
```
There's a `/` at the beginning, right? It was there before.

Reply:
Absolutely! I seem to have missed it. Have added it back now.